build/gulp/tasks/generate/templates/docs-examples-components/$DisplayName/Types/$DisplayNameExample.shorthand.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { $DisplayName } from '@stardust-ui/react'
 
-const $DisplayNameExampleShorthand = () => <$DisplayName />
+const $DisplayNameExampleShorthand = () => <$DisplayName content="Hello from $DisplayName!" />
```

Comment:
Freebie.  The initial component example from a generated component had no content.  Added some here so something shows up in the docs.